.github/workflows/build_brltty_windows.yml

```diff
@@ -0,0 +1,88 @@
+name: build BRLTTY for Windows
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    branches: [master]
+
+  workflow_dispatch:
+
+jobs:
+  build_brltty:
+    runs-on: windows-2025
+
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.13
+          architecture: x64
+      - name: Install setuptools and cython
+        shell: cmd
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools cython
+
+      - name: Set up MSYS environment
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: UCRT64
+          update: true
+          install: >-
+            autoconf
+            automake
+            make
+            awk
+            bison
+            git
+            groff
+            m4
+            patch
+          pacboy: >-
+            tcl:p
+            pdcurses:p
+            icu:p
+            tcl:p
+            libiconv:p
+            toolchain:p
+            libusb:p
+
+      - name: Apply patches
+        run: patch -p0 <"Windows/affinity.patch"
+
+      - name: Configure build
+        run: |
+          export PATH="$(cygpath -u "$pythonLocation")/scripts:$(cygpath -u "$pythonLocation"):$PATH"
+          ./autogen
+          ./cfg-windows --prefix=/ --enable-relocatable-install --with-usb-package=winusb
```

Comment:
Depending on what the goal of the builds is, you might want to set up a matrix wit certain variants. The official builds offer libusb/winusb variants and are 32 bits for now, so we might want to offer a 32 bits build as well if there is still any interest in that. Also, the Python bindings are now just for Python x64, you could also add multiple Python versions there.